intelmq/bots/experts/sieve/expert.py

```diff
@@ -230,6 +233,14 @@ def process_ip_range_match(self, key, ip_range, event):
                 if addr in network:
                     return True
         return False
+    
+    @staticmethod
+    def compute_basic_math(action, event):
+        date = dateutil.parser.parse(event[action.key])
```

Comment:
The event should have isoformat only, so no special parsing should be needed here.
See `DateTime.parse_utc_isoformat`